### DISCUSSION TITLE
Will security vulnerability fix be ported to .net standard version?

### DISCUSSION THREAD

**Discussion by c0bra99** [SixLabors/ImageSharp#2684] [Q&A] [answered]
The security vulnerability is only fixed in 3.1.3, but this version does not support .net standard. 

Will the vulnerability be fixed in a version that supports .net standard?

**Comment by JimBobSquarePants** [comment]
There are currently no plans to backport the fix. V3 has been out now for over 1 year and most of the target frameworks that are supported by V2 are not supported by MS.

**Comment by c0bra99** [comment, marked as answer]
Version 2.1.7 was just released that fixed the vulnerability for .net standard. Thank you, this means we do not have to stop using ImageSharp
